staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/validation/validation.go

```diff
@@ -67,11 +67,17 @@ func convertJSONSchemaProps(in *apiextensions.JSONSchemaProps, out *spec.Schema)
 	out.MinItems = in.MinItems
 	out.UniqueItems = in.UniqueItems
 	out.MultipleOf = in.MultipleOf
-	out.Enum = in.Enum
 	out.MaxProperties = in.MaxProperties
 	out.MinProperties = in.MinProperties
 	out.Required = in.Required
-	out.Example = in.Example
+	out.Default = interface{}(in.Default)
```

Comment:
Is this actually necessary to type-cast?

Reply:
Will try removing it.

Reply:
and is it what we want? Does it store the type of the json data structure (string, bool, int, float, map, slice) in `Default` or `JSON(...)`?

Reply:
needs a comment at least